lib/matplotlib/collections.py

```diff
@@ -80,9 +80,7 @@ class Collection(artist.Artist, cm.ScalarMappable):
     # _offsets must be a Nx2 array!
     _offsets.shape = (0, 2)
     _transOffset = transforms.IdentityTransform()
-    _transforms = []
-
-
+    _transforms = np.empty((0, 3, 3))
```

Comment:
I see with `git grep` that there is a small number of `#:` comments in the code, but that's not a very easy thing to search. Does Sphinx do something special with them, or some other tool?